### PR TITLE
AMP-63922 Adding base64 note

### PR DESCRIPTION
**Pull request by caseyamp**
# Amplitude Developer Docs PR

Adds note about authorization when using cURL

## Deadline

ASAP

## Change type

- [X] Doc update.

# PR checklist:

- [X] My documentation follows the style guidelines of this project.
- [X] I previewed my documentation on a local server using `mkdocs serve`.
- [X] Running `mkdocs serve` didn't generate any failures.
- [X] I have performed a self-review of my own documentation.

### CODE REVIEW

File: docs/analytics/apis/user-privacy-api.md [hunk 1]
```diff
@@ -70,7 +76,7 @@ The body parameter is required. It's the deletion request object listing the `us
 
     ```bash
     curl --location --request POST 'https://amplitude.com/api/2/deletions/users' \
-    --header 'Authorization: Basic {{api-key}}:{{secret-key}} \
+    --header 'Authorization: Basic {{api-key}}:{{secret-key}} \ # API Key:Secret Key must be Base64-encoded
```
Comment (LeontiBrechko) [inline docs/analytics/apis/user-privacy-api.md:79]: I love the `# API Key:Secret Key must be Base64-encoded` comment!

How about putting something like:
`--header 'Authorization: Basic {{base64-encoded-credentials}} \ # "{{api-key}}:{{secret-key}}" pair must be Base64-encoded`

Also, let's make similar changes in other places, which mention `Authorization: Basic {{api-key}}:{{secret-key}}`

File: docs/analytics/apis/user-privacy-api.md [hunk 2]
```diff
@@ -11,6 +11,12 @@ The User Privacy API helps you comply with end-user data deletion requests manda
 
 --8<-- "includes/auth-basic.md"
 
+!!!note "Authorization in cURL"
```
Comment (LeontiBrechko) [inline docs/analytics/apis/user-privacy-api.md:14]: Sorry for missing this note in the ticket, but I believe this applies to all communication methods, which specify the header directly, not only cURL

I would recommend a more generic statement (feel free to use different wording =)):
```
!!!note "Setting Authorization Header"

When specifying the `Authorization: Basic` header directly, the `acesss_key:secret_key` credentials pair should be Base64-encoded. It should look something like this:
```

File: docs/analytics/apis/user-privacy-api.md (hunk 1 above)
Comment (caseyamp) [reply docs/analytics/apis/user-privacy-api.md:79]: This is my fault -- I wrongly assumed that anyone working with the API would know that basic auth requires base64 encoding of creds. Forgive me. SO, this needed to be added to the Basic Auth reusable snippet we use for every API that uses Basic Auth see #430  for that work.

File: docs/analytics/apis/user-privacy-api.md (hunk 2 above)
Comment (caseyamp) [reply docs/analytics/apis/user-privacy-api.md:14]: 🤦 yes. I was not firing on all cylinders yesterday, thank you.

**Pull request merged**